admin/acceptance.py

```diff
@@ -163,12 +163,14 @@ class VagrantRunner(object):
     # rather than assuming it is available.
     # https://clusterhq.atlassian.net/browse/FLOC-1163
 
-    NODE_ADDRESSES = ["172.16.255.240", "172.16.255.241"]
+    NODE_ADDRESSES = ["172.16.255.250", "172.16.255.251"]
```

Comment:
No. Since the acceptance tests and getting started guide share the control node certificate, it has to have the same address.